Dockerfile

```diff
@@ -16,7 +16,9 @@ ENV CMK_SITE=${CMK_SITE_ARG}
 ENV CMK_PASSWORD=${CMK_PASSWORD_ARG}
 ENV MAILHUB=${MAILHUB}
 
-RUN yum -y install epel-release && yum update -y
+VOLUME ["/opt/backup", "/opt/omd/sites"]
+
+RUN yum -y install epel-release && yum update -y && yum install openssh-clients
```

Comment:
This doesn't work because the build is not interactive --> you have to use `yum -y install`
But please add the `openssh-clients` to the `yum install` some lines down where there are already some additional packets listed (ssmtp, which)

Reply:
Yep I figured the -y later on. I'm gonna correct that, however I'm unsure as to how to add these changes to the pull request. Let me try to figure it out